website/static/css/site.css

```diff
@@ -1406,6 +1393,27 @@ dl.activity-log dd {
     width: 400px;
 }
 
+/* Citations */
```

Comment:
We have begun to modularize our CSS, using smaller CSS files and  importing them where needed. You can put this CSS in its own file, e.g. `citations.css` and then import it in `citations.js` using `require('../css/citations.css')`.